docs/How-We-Work/team-charter/team-meetings.md

```diff
@@ -1,74 +1,66 @@
 # Meeting Norms 
 
-- For all recurring team meetings and standups use OFA WebEx in the invite. Zoom or Google Meet is okay for ad hoc conversations. 
+- For all recurring team meetings and standups use Zoom.
 - If you have background noise, mute your mic. 
 - Try not to talk over others.  
-- If you are creating a new meeting invite, please be explicit include an goal/outcome in the meeting invite. What do we hope to accomplish in that time? 
-- OFA and Raft both use Outlook and 18F uses Google Calendar. Generally, cross-platform calendar invites work okay. However, if you modify an existing meeting--especially a recurring meeting--double check that the invitees got the update. 
+- If you are creating a new meeting invite, please be explicit and include an goal/outcome and/or an agenda in the meeting invite. What do we hope to accomplish in that time? 
+- Be open and receptive to others' ideas and contributions.
+- No meetings on Thursdays as much as possible
+- Cameras on in retros and team happy hours / Cameras optional, but encouraged, in other meetings. Please be mindful that some team members work better by reading non-verbal communication of other team members.
+- If there are action items that come out of a meeting, document and follow-up.
+- Meeting notes should be stored in a [shared location](https://hhsgov.sharepoint.com/sites/TANFDataPortalOFA/_layouts/15/Doc.aspx?sourcedoc=%7Bcbce2e75-17b2-4e70-b422-60d034fcd4af%7D&action=edit&wd=target%28General.one%7C76fa641e-4d01-436b-a079-42ae19270071%2FGeneral%20Links%7C38eda373-ae66-41ed-9d2d-e7b010d5b312%2F%29).
 
 
 # Recurring Meetings 
 
 ## Standup 
 - Goal: To inspect progress towards sprint goal and identify blockers
 - Attendees: Core team 
-- Facilitator: None 
-- When: Daily 10:30-10:45am EST and asynchronously on Wednesdays
+- Facilitator: Valerie
+- When: Daily 12pm-12:15pm EST and asynchronously on Thursdays and every other Tuesday when the team has sprint ceremonies
 - Format: Each team member gives quick update to answer these questions 
   - *What did you do yesterday that advances the sprint goal?* 
```

Comment:
```suggestion
  - Facilitator shares their screen and pulls up assigned tickets for each team member in the current sprint. 
  - *What did you do since the last standup that advances the sprint goals?* 
```